ERCS/erc-7390.md

```diff
@@ -0,0 +1,364 @@
+---
+eip: 7390
+title: Vanilla Option Standard
```

Comment:
How about:

```suggestion
title: Vanilla Options for ERC-20 Tokens
```

I think that's a bit more descriptive?

Reply:
Yes you're right, we where maybe planning to add support for other token but I'd be a more sophisticated codebase with exceptions etc... Not sure if it will be implemented.